### PR TITLE
Adding - Mark Mandel

### PR DESCRIPTION
**Pull request by markmandel**
Adding myself to this awesome list! :+1:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ Like `awesome-developer-streams`? Reach out to [@bitandbang](https://twitter.com
 - [Luke Gorrie](#luke-gorrie) - **streaming:** C, Lua, RaptorJIT, R, Nix, Network drivers
 - [Marco Bruno](#marco-bruno) - **streaming** Programming, Javascript, Web Development, React, CSS, HTML
 - [Mark Kraus](#mark-kraus) - **streaming** PowerShell, C#, Azure Functions, CI/CD Pipelines, OSS
+- [Mark Mandel](#mark-mandel) - **streaming** Game Deveopment, Kubernetes, Go, OSS
```
Comment (mblayman) [inline README.md]: There's a typo in Development.

File: README.md [hunk 2]
```diff
@@ -906,6 +907,20 @@ Web Development, React, Node.js, CSS, Next.js, Wordpress
 - [Github](https://github.com/mblayman)
 - [YouTube Playlist](https://www.youtube.com/watch?v=fNEZ_1dmVyE&list=PLFcKEo4b_n1wQA6lKtSqCq1dTMq1c2Lmw)
 
+### Mark Mandel
```
Comment (mblayman) [inline README.md]: This section needs to go above Matt Layman in order to remain alphabetically sorted. The line added in the index is already correct.

**Comment by markmandel** [comment]
Oops! Thanks for review, will make the changes!

**Comment by markmandel** [comment]
...and done! :+1:

**Pull request merged**